packages/terra-icon/tests/wdio/icon-spec.js

```diff
@@ -20,4 +20,11 @@ describe('Icon', () => {
     Terra.should.beAccessible();
     Terra.should.matchScreenshot();
   });
+
+  describe('AriaLabel', () => {
+    before(() => browser.url('/#/raw/tests/terra-icon/icon/icon-aria-label'));
```

Comment:
Does this warrant a new screenshot test? All we are really doing is adding aria-label to the alert, which would make beAccessible pass whether it's set or not.

Reply:
Yeah, let's drop this line:

```
 Terra.should.matchScreenshot();
```